src/locator_strict.rs

```diff
@@ -67,10 +81,18 @@ macro_rules! strict {
 /// Packages may also be namespaced to a specific organization;
 /// in such cases the organization ID is at the start of the `{package}` field
 /// separated by a slash. The ID can be any non-negative integer.
-/// This yields the following format:
+/// This yields the following optional format:
```

Comment:
[Question] What is "optional" in this context?

Reply:
Meaning, "you can optionally provide this format along with the formats already documented above"